src/vcpkg/commands.search.cpp

```diff
@@ -54,10 +55,18 @@ namespace vcpkg::Commands::Search
             {
                 description = source_paragraph.description[0];
             }
-            System::printf("%-20s %-16s %s\n",
-                           vcpkg::shorten_text(source_paragraph.name, 20),
-                           vcpkg::shorten_text(full_version, 16),
-                           vcpkg::shorten_text(description, 81));
+            static constexpr const int name_columns = 24;
```

Comment:
Won't `azure-security-keyvault-common-cpp` go over 24 (it's 34).
Where's the version string for that one?

Along with that, it's possible the version string could be ~19 chars:
99.99.99-beta.99#99

So, 34+19+1 (for space) = 55.
How does that relate to 24 or 41 for `s_name_and_ver_columns` above?

Reply:
Oh, looks like you are using `max` of space remaining and the actual size, so that makes it all work out well.